parmed/rdkit/rdkit.py

```diff
@@ -32,10 +32,22 @@ def load(rmol):
         return PDBFile.parse(fh)
 
     @staticmethod
-    def from_smiles(smiles):
+    def from_smiles(smiles, coordinates=True):
         """
         Load smiles string to :class:`Structure`
+
+        Parameters
+        ----------
+        smiles : str, smiles
+        coordinates : bool, default True
+            if True, use `rdkit.Chem.AllChem.EmbedMultipleConfs to assign coordinates
+            if False, coordinates for all atoms are all 0.0
```

Comment:
Agree.  I actually tested "None" but failed. Lol.

I will update the code to None

Hai

> On Jun 10, 2016, at 7:26 PM, Jason Swails <notifications@github.com> wrote:
> 
> In parmed/rdkit/rdkit.py:
> 
> >          """
> >          Load smiles string to :class:`Structure`
> > +
> > +        Parameters
> > +        ----------
> > +        smiles : str, smiles
> > +        coordinates : bool, default True
> > +            if True, use `rdkit.Chem.AllChem.EmbedMultipleConfs to assign coordinates
> > +            if False, coordinates for all atoms are all 0.0
> Ah got it. I think None is more consistent overall.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>